lacci/lib/shoes/drawables/arrow.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+module Shoes
+  class Arrow < Shoes::Drawable
+    shoes_style :draw_context
+
+    [:left, :top, :width].each do |prop|
+      shoes_style(prop) { |val| val.is_a?(Hash) ? val : convert_to_integer(val, prop) }
+    end
+
+    def initialize(*args)
+      @draw_context = Shoes::App.instance.current_draw_context
+
+      super
+
+      if args.length == 1 && args[0].is_a?(Hash)
+        options = args[0]
+        self.left = options[:left]
```

Comment:
If you pass left, top and width as keyword args, the super call should set them automatically.